testsuite/testsuite-webservices/pom.xml

```diff
@@ -20,13 +20,34 @@
   <name>Test Suite: WebServices</name>
   <description>Test Suite: WebServices</description>
 
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.19.1</version>
+        <configuration>
+          <argLine>-Xbootclasspath/p:${user.home}/.m2/repository/org/jboss/logmanager/jboss-logmanager/2.0.3.Final/jboss-logmanager-2.0.3.Final.jar
```

Comment:
Because depending on how I set things up, I would get complaints from jboss-logging. It seems redundant right now though